website/src/components/features.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import Layout from '@theme/Layout';
+import BasicInstall from "../components/basicInstall"
+
+const Feature = (props) =>
+  <div className="row padding--md feature-row">
+    <div className="col col--1"/>
+    <div className="col col--6" key="img">
+      {!props.image ? "" : <img src={props.image}/>}
+    </div>
+    <div className="col col--5" key="text">
+      <div className="padding--md"/>
+      <h3>{props.title}</h3>
+      <div className="padding--sm"/>
+      {props.children}
+    </div>
+    <div className="col col--1"/>
+  </div>
+
+const FeatureList = [
+    <Feature image="img/envs.gif" title="Scala versions, dependencies and JVMs" 
+      key="versions" education="true" simple_projects="true" scripting="true" prototyping="true"> 
+      <p>
+        scala-cli is built on top of coursier
+        <br/>
+        This allow us to manage Scala versions, dependencies and JVMs so you can test your code in different environments by changing single option.
+      </p>
+
+      <p>
+        scala-cli install all its depenendencies
```

Comment:
?
```suggestion
        Scala CLI installs all its dependencies
```

"ships" sounds better too, although, strictly speaking, Scala CLI installs / downloads them on-the-fly rather than being shipped with them…
```suggestion
        Scala CLI ships with all its dependencies
```